src/magnify/apps/core/urls.py

```diff
@@ -13,15 +16,25 @@
         title="Magnify API",
         default_version="v1",
         description="""This is the schema for the Magnify API.
-            This app is used in the sandbox folder of the project.""",
+
+            All routes are protected by the JWT authentication except
+            for "login", "refresh" and "create user".""",
     ),
     public=True,
+    url=os.environ.get("BACK_URL"),
```

Comment:
you should go through a Django setting (see existing settings with value.Value())